### PR TITLE
Add confirmed_at to swap info

### PR DESCRIPTION
**Pull request by ademar111190**

### CODE REVIEW

**Review by roeierez — APPROVED** [review]
LGTM

**Review by dangeross — COMMENTED** [review]
I think the PR title is misleading. I see confirmed_at added to the swaps_info table, but created_at not removed from the swaps table. Is that the intent?

File: libs/sdk-core/src/models.rs [hunk 1]
```diff
@@ -1288,6 +1288,8 @@ pub struct SwapInfo {
     ///
     /// Swaps created after dynamic fees were introduced always have this field set.
     pub channel_opening_fees: Option<OpeningFeeParams>,
+    /// The block height when the swap was confirmed.
+    pub confirmed_at: u32,
```
Comment (dangeross) [inline libs/sdk-core/src/models.rs:1292]: Would it be better for it to be an Option<u32> and nullable in the table?

**Comment by roeierez** [comment]
> I think the PR title is misleading. I see confirmed_at added to the swaps_info table, but created_at not removed from the swaps table. Is that the intent?

Yes the intent is only add a field as the created_at is still valid (need some fix in a different PR).

File: libs/sdk-core/src/models.rs (hunk 1 above)
Comment (roeierez) [reply libs/sdk-core/src/models.rs]: Yes, makes sense. Donw

**Review by dangeross — APPROVED** [review]
LGTM

File: libs/sdk-core/src/swap_in/swap.rs [hunk 2]
```diff
@@ -381,14 +383,16 @@ impl BTCReceiveSwap {
...
-            utxos.confirmed_sats(),
-            utxos.confirmed_tx_ids(),
-            swap_status,
-        )?)
+        let chain_info = SwapChainInfo {
+            unconfirmed_sats: utxos.unconfirmed_sats(),
+            unconfirmed_tx_ids: utxos.unconfirmed_tx_ids(),
+            confirmed_sats: utxos.confirmed_sats(),
+            confirmed_tx_ids: utxos.confirmed_tx_ids(),
+            confirmed_at: confirmed_block,
```
Comment (roeierez) [inline libs/sdk-core/src/swap_in/swap.rs:391]: I see a bug here where the confirmed_at is not optional and may be zero.

Comment (dangeross) [reply libs/sdk-core/src/swap_in/swap.rs:391]: Good catch, I thought at this point the known

**Review by dangeross — APPROVED** [review]

**Review by ok300 — APPROVED** [review]

**Pull request merged**